modules/social_features/social_event/modules/social_event_managers/src/SocialEventManagersAccessHelper.php

```diff
@@ -8,55 +8,80 @@
 
 /**
  * Helper class for checking update access on event managers nodes.
+ *
+ * @todo Check: probably, these access rules should be applied only for events
+ *   with not empty "field_event_managers" field.
  */
 class SocialEventManagersAccessHelper {
 
+  /**
+   * Neutral status.
+   */
+  const NEUTRAL = 0;
```

Comment:
nice change, don't forget the visibility of the const.

Reply:
@tbsiqueira We don't need it - https://github.com/goalgorilla/open_social/pull/3522/files#diff-ca4cf6b9cb9a70e2d7933a10c6009542b4f31e079627577c6b077aad9a63a14dL63

Reply:
@tbsiqueira added